devel/api.md

```diff
@@ -47,7 +47,14 @@
     * [`Ethernet.DUPLEX`](#ethernetduplex)
     * [`Ethernet.SPEED`](#ethernetspeed)
     * [`Ethernet.SRIOV_SUBTREE`](#ethernetsriov)
-	* [`Ethernet.SRIOV.TOTAL_VFS`](#sriovtotal_vfs)
+        * [`Ethernet.SRIOV.TOTAL_VFS`](#sriovtotal_vfs)
```

Comment:
The anchors need to be the full constant name with dots removed. The unchanged one for the subtree is wrong, too.